routersploit/interpreter.py

```diff
@@ -39,8 +39,10 @@
 
 
 def is_libedit():
-    return "libedit" in readline.__doc__
-
+    try:
+        return "libedit" in readline.__doc__
+    except TypeError:
+        return False
```

Comment:
In my view this style a little bit better
```python
return isinstance(readline.__doc__, str) and "libedit" in readline.__doc__
```

Reply:
I added your style fix into a commit